tools/accuracy_checker/accuracy_checker/config/config_reader.py

```diff
@@ -202,9 +172,8 @@ def _check_module_config(config):
         eval_mode = get_mode(config)
         config_checker_func = config_checkers.get(eval_mode)
         if config_checker_func is None:
-            raise ConfigError('Accuracy Checker {} mode is not supported. Please select between {}'. format(
-                eval_mode, ', '.join(['evaluations', 'models', 'pipelines'])
-            ))
+            raise ConfigError('Accuracy Checker {} mode is not supported. Please select between {} and {}.'. format(
+                eval_mode, 'evaluations', 'models'))
```

Comment:
Substituting constant strings doesn't make sense - just embed them in the format string.

Reply:
agrree, fixed